SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
+
+## Supported Versions
+
+Maintained versions are per [Maintaince Policy](https://mariadb.org/about/#maintenance-policy). This will correspond to the major version number directories in this repository.
+
+## Reporting a Vulnerability
+
+The Docker Official Image of MariaDB Server includes binaries from a number of sources:
+* `gosu` from https://github.com/tianon/gosu;
+* the base container, i.e. Ubuntu;
+* `docker-entrypoint.sh`/build and `healthcheck.sh` scripts; and
+* MariaDB upstream packages.
+
+`gosu`, based on the upstream [security vulnerability reporting](https://github.com/tianon/gosu/security/advisories/new), should be validated using [govulcheck](https://pkg.go.dev/golang.org/x/vuln/cmd/govulncheck) to see if any CVE within these libraries are actually used by the `gosu` executable. This container can pick up a new `gosu` version after there is a upstream release.
+
+The base image of MariaDB Server is based on other Docker Official Images, which are [periodly updated](https://github.com/docker-library/official-images/commits/master/library/ubuntu). When the base Docker Official Image is updated, the MariaDB is [also updated](https://github.com/docker-library/repo-info/commits/master/repos/mariadb). Should a freshly pulled MariaDB current image be affected by a vulnerability of its base image, please do a [vulnerability report with Docker Official Images](https://github.com/docker-library/official-images/security/advisories/new).
```

Comment:
Can this perhaps link to https://github.com/docker-library/faq#why-does-my-security-scanner-show-that-an-image-has-cves and/or https://github.com/docker-library/official-images/blob/master/SECURITY.md instead? :sweat_smile:

We absolutely 100% do _not_ want folks to run some security scanner and then paste the result in a security advisory to us because that's a waste of everyone's time (and avoiding this is partly I'm guessing what you're trying to avoid for yourself too with this document). :grimacing:

Reply:
Good docs. Thanks, added. I see you've been hit with this a few times before :smiling_face_with_tear: .

Yes, absolutely trying to avoid the next "security scanner says bad" reports and allow genuine reports to be received by the relevant people.